src/prop-types/ref.js

```diff
@@ -1,5 +1,5 @@
 import PropTypes from 'prop-types';
 
 export default PropTypes.shape({
-  value: PropTypes.any,
+  current: PropTypes.any,
```

Comment:
This didn't previously through any prop type warnings because the `value` property is not required. As long as the prop is an object, this won't cause any warnings.

Reply:
Also, if the `current` property isn't required, then it doesn't really make sense to use `shape`, and we should use `PropTypes.object` instead. Should have the same result.